Jenkinsfile

```diff
@@ -159,16 +159,16 @@ pipeline {
                     }
                     post { always { retry(3) { deleteDir() } } }
                 }
-                stage('GPU Tests') {
-                    agent { label "gpu" }
+                stage('OpenCL Tests') {
+                    agent { label "opencl" }
```

Comment:
we probably want to keep this label as 'GPU' because it specifies that we want a node with a GPU